agent/collector/ingester.go

```diff
@@ -0,0 +1,130 @@
+package collector
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	"github.com/kubeshop/tracetest/server/otlp"
+	pb "go.opentelemetry.io/proto/otlp/collector/trace/v1"
+	v1 "go.opentelemetry.io/proto/otlp/trace/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type ingester interface {
```

Comment:
this interface also seems unused, maybe it can be removed?

Reply:
changed it to `stoppable` because we might have to stop the ingester.